### PR TITLE
Split faq into sections

### PR DESCRIPTION
**Pull request by tsetsova**
Hi, 

This is in response to issue #342 

Here's a screenshot of what the change looks like:

screenshot from 2018-08-14 21-17-12

Let me know if you need anything else before you can merge it in.

### CODE REVIEW

File: app/views/home/faq.html.erb [hunk 1]
```diff
@@ -2,21 +2,30 @@
   <h1 class="page-title">FAQ</h1>
 
   <ul>
+    <h3 class='faq-section-header'><%= link_to "General", anchor: "general_header" %></h3>
     <li><%= link_to "What is diversitytickets.org?", anchor: "general_org" %></li>
```
Comment (guainumbi) [inline app/views/home/faq.html.erb]: Example: `<h3 class='faq-section-header'><%= link_to t('.general'), anchor: "general_header" %></h3>`

**Comment by guainumbi** [comment]
hi @tsetsova :) We haven't heard from you in a while. Please let us know if you would like to make the changes to your PR yourself. We would love to add your changes to the next deployment, but the translation links need to be included for this. If you can not find the time we can also merge your contributions to another branch and adapt the necessary changes for you. This way your previous commits will still count and be honoured during deployment.

**Comment by tsetsova** [comment]
Hi I’ve been away sorry! 

I can take a look in the next couple of weeks.

Sorry for the delay

**Comment by tsetsova** [comment]
Hi @guainumbi,
I've changed the base to staging and added the localisation changes.

Let me know if you need anything else.

Thanks!

**Pull request merged**